### DISCUSSION TITLE
Change Default Text in ManimGL

### DISCUSSION THREAD

**Discussion by Othello14** [3b1b/manim#1507] [Q&A] [answered]
How do I change the default text in ManimGL? I want to transform from a math tex to a decimal number but they have different fonts so it looks weird. Alternatively, how could I change the font of the decimal number, below doesn't seem to change anything:

`        decimal1 = DecimalNumber(
                2,
                num_decimal_places=2,
                include_sign=False,
                unit="^x",
                color=BLUE,
                font="Arial"
            )`

**Comment by TonyCrane** [comment, marked as answer]
Change `style.font` in the config file (.yml).
